ui/react-components/components/AppointmentEditorFooter/AppointmentEditorFooter.module.scss

```diff
@@ -5,7 +5,6 @@
   height: 50px;
   padding: 0 32px;
   position: absolute;
-  bottom: 0;
```

Comment:
I think this is required to keep the footer down

Reply:
its again footer issue, if its there, its messing while recurring is in